src/platform/site-wide/cta-widget/ctaWidgets.js

```diff
@@ -141,6 +143,17 @@ export const ctaWidgetsLookup = {
     requiredServices: null,
     serviceDescription: 'order hearing aid supplies',
   },
+  [CTA_WIDGET_TYPES.HIGHER_LEVEL_REVIEW]: {
+    id: CTA_WIDGET_TYPES.HIGHER_LEVEL_REVIEW,
+    deriveToolUrlDetails: () => ({
+      url: higherLevelReviewUrl,
+      redirect: false,
+    }),
+    isHealthTool: false,
+    mhvToolName: null,
+    requiredServices: null,
+    serviceDescription: 'request a Higher-Level Review',
```

Comment:
Should Higher-Level Review be capitalized in content?

Reply:
Yes, the form name is always captialized